stripe/src/main/java/com/stripe/android/model/Stripe3ds2Fingerprint.kt

```diff
@@ -78,20 +78,16 @@ internal class Stripe3ds2Fingerprint private constructor(
 
         @JvmSynthetic
         @Throws(CertificateException::class)
-        internal fun create(sdkData: StripeIntent.SdkData): Stripe3ds2Fingerprint {
-            require(sdkData.data is StripeIntent.NextActionData.SdkData.`3DS2`) {
-                "Expected SdkData with type='stripe_3ds2_fingerprint'."
-            }
-
+        internal fun create(sdkData: StripeIntent.NextActionData.SdkData.`3DS2`): Stripe3ds2Fingerprint {
```

Comment:
optional: make this a secondary constructor instead of a `companion object` method